packages/appstore/src/components/main-title-bar/asset-summary.tsx

```diff
@@ -6,7 +6,46 @@ import BalanceText from 'Components/elements/text/balance-text';
 import { observer, useStore } from '@deriv/stores';
 import './asset-summary.scss';
 import TotalAssetsLoader from 'Components/pre-loader/total-assets-loader';
-import { useTotalAccountBalance, useCFDAccounts, usePlatformAccounts } from '@deriv/hooks';
+import {
+    useTotalAccountBalance,
+    useCFDAccounts,
+    usePlatformAccounts,
+    useTotalAssetCurrency,
+    useExchangeRate,
+} from '@deriv/hooks';
+
+const isRatesLoaded = (
```

Comment:
can you move this to a separate file/hook? best to keep the logic separated from the components.